index.js

```diff
@@ -5,7 +5,7 @@ var SETTLED = 'settled';
 var FULFILLED = 'fulfilled';
 var REJECTED = 'rejected';
 var NOOP = function () {};
-var isNode = typeof process !== 'undefined' && typeof process.emit === 'function';
+var isNode = global.process !== 'undefined' && typeof global.process.emit === 'function';
```

Comment:
woops. should have been typeof global.process or undefined should not be a string